lib/hooks.js

```diff
@@ -127,34 +128,44 @@ const usePlaybackStateIs = (...states) => {
  *   (interval: number) => void
  * ]}
  */
-const useTrackPlayerProgress = (interval = 1000) => {
+const useTrackPlayerProgress = (
+    interval = 1000,
+    pollTrackPlayerStates = [
+      TrackPlayer.STATE_PLAYING,
+      TrackPlayer.STATE_BUFFERING,
+    ]
+  ) => {
     const initialState = {
-        position: 0,
-        bufferedPosition: 0,
-        duration: 0
+      position: 0,
+      bufferedPosition: 0,
+      duration: 0,
     };
 
     const [state, setState] = useState(initialState);
 
     const getProgress = async () => {
-        const [position, bufferedPosition, duration] = await Promise.all([
-            TrackPlayer.getPosition(),
-            TrackPlayer.getBufferedPosition(),
-            TrackPlayer.getDuration()
-        ])
-        setState({ position, bufferedPosition, duration });
+      const [position, bufferedPosition, duration] = await Promise.all([
+        TrackPlayer.getPosition(),
+        TrackPlayer.getBufferedPosition(),
+        TrackPlayer.getDuration(),
+      ]);
+      setState({ position, bufferedPosition, duration });
+    };
```

Comment:
The whole file is indented with 4 spaces, can you keep the same code style?

Reply:
Ok, I indented the example.